<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmsGsmOptionsWidget</class>
 <widget class="QWidget" name="SmsGsmOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>311</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>SMS: GSM</string>
  </property>
  <widget class="QGroupBox" name="gpPrimaryGsmModem">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>15</y>
     <width>280</width>
     <height>200</height>
    </rect>
   </property>
   <property name="title">
    <string> Primary GSM modem  </string>
   </property>
   <widget class="QComboBox" name="cmbPrimaryComPort">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>160</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>COM1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM4</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cmbPrimaryBaudRate">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>160</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>110</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>300</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>56000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>128000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>256000</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cmbPrimaryModemType">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>160</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Siemens A1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Siemens M1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Siemens M20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ericsson GM12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nokia 6210</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nokia 6650</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nokia 9110</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Siemens M35</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Siemens S25</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Motorola T260</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>WaveCom</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Siemens TC35</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Wavecom Fasttrack</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Z-text SMS modem</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ericsson K750i</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ericsson W660i</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Falcom Samba 75</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Huawei Connect 3G</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Huawei E220</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Maestro 20 GSM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Motorola G20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MultiTech MTCBA-E</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MultiTech MTCBA-G-F1-EU</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nokia 6310i</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nokia N30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Siemens ES75</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cmbPrimaryMode">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>160</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>PDU</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Text</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="editPrimaryPin">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>160</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editPrimaryServiceCenter">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>160</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblPrimaryComPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Com Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPrimaryBaudRate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>55</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Baud rate</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPrimaryModemType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>85</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Modem Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPrimaryMode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>115</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPrimaryPin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>145</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>PIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPrimaryServiceCenter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>175</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Service Center</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gpBackupGsmModem">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>15</y>
     <width>280</width>
     <height>210</height>
    </rect>
   </property>
   <property name="title">
    <string> Backup GSM modem  </string>
   </property>
   <widget class="QLineEdit" name="editBackupPin">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>160</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editBackupServiceCenter">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>160</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbBackupModemType">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>160</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Siemens A1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Siemens M1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Siemens M20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ericsson GM12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nokia 6210</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nokia 6650</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nokia 9110</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Siemens M35</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Siemens S25</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Motorola T260</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>WaveCom</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Siemens TC35</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Wavecom Fasttrack</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Z-text SMS modem</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ericsson K750i</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ericsson W660i</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Falcom Samba 75</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Huawei Connect 3G</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Huawei E220</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Maestro 20 GSM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Motorola G20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MultiTech MTCBA-E</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MultiTech MTCBA-G-F1-EU</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nokia 6310i</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nokia N30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Siemens ES75</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lblBackupPin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>145</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>PIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBackupModemType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>85</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Modem Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBackupServiceCenter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>175</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Service Center</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBackupMode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>115</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBackupComPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Com Port</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbBackupMode">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>160</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>PDU</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Text</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cmbBackupComPort">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>160</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>COM1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM4</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cmbBackupBaudRate">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>160</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>110</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>300</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>56000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>128000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>256000</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lblBackupBaudRate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>55</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Baud rate</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblSmsLog">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>225</y>
     <width>65</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>SMS Log:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblLogFile">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>255</y>
     <width>65</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Log file:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editLogFile">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>168</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbSmsLog">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>195</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>No log</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Failed events only</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Failed and successful events</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Detailed tech info</string>
    </property>
   </item>
  </widget>
  <widget class="QToolButton" name="btnSelectLogFile">
   <property name="geometry">
    <rect>
     <x>248</x>
     <y>250</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Log file</string>
   </property>
   <property name="icon">
    <iconset resource="../../qHostMonitor.qrc">
     <normaloff>:/img/open.png</normaloff>:/img/open.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTestGsmModem">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>235</y>
     <width>110</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/resources/RC Data/0000/TOPTIONSFORM/gsmTestBtn.Glyph.Data.png</normaloff>:/resources/RC Data/0000/TOPTIONSFORM/gsmTestBtn.Glyph.Data.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../qHostMonitor.qrc"/>
  <include location="../../qHostMonitor.qrc"/>
 </resources>
 <connections/>
</ui>
